__tests__/server.test.js

```diff
@@ -105,6 +105,24 @@ describe("GET /api/articles", () => {
         expect(body.articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("should return only the articles that have the correct topics according to the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("should return a 404 error if there are no articles for that topic", () => {
```

Comment:
this test description isn't really what you're testing, you're checking here that a valid but non-existent topic gives you a 404 which is correct, just update the description to be a bit clearer